xray/src/generation.rs

```diff
@@ -154,6 +154,8 @@ pub trait ColoringStrategy: Send {
     // After all points are processed, this is used to query the color that should be assigned to
     // the pixel (x, y) in the final tile image.
     fn get_pixel_color(&self, x: u32, y: u32, background_color: Color<u8>) -> Color<u8>;
+
+    fn attributes(&self) -> Vec<String>;
```

Comment:
`'static str` for avoiding `to_string()`?

Reply:
Good idea, changed it.